api-server/ecosystem.config.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const path = require('path');
+
+const dotenv = require('dotenv');
+
+const filePath = path.resolve('..', '.env');
+const env = dotenv.parse(fs.readFileSync(filePath));
+
+module.exports = {
+  apps: [
+    {
+      script: `./lib/production-start.js`,
+      env: { ...env, DEBUG: 'fcc*' },
```

Comment:
```suggestion
      env,
```

I really really really want to enable logs on demand.

Reply:
Sure thing! Not sure what I was thinking there.